### PR TITLE
osd/ReplicatedBackend: remove useless assert

### PR DESCRIPTION
**Pull request by majianpeng**

### CODE REVIEW

**Comment by batrick** [comment]
retest this please

**Comment by tchaikov** [comment]
@majianpeng could you explain the rationales behind your change in the commit message?

**Comment by tchaikov** [comment]
retest this please.

File: src/osd/ReplicatedBackend.cc [hunk 1]
```diff
@@ -1103,7 +1105,7 @@ void ReplicatedBackend::repop_commit(RepModifyRef rm)
     get_parent()->whoami_shard(),
     0, get_osdmap()->get_epoch(), m->get_min_epoch(), CEPH_OSD_FLAG_ONDISK);
   reply->set_last_complete_ondisk(rm->last_complete);
-  reply->set_priority(CEPH_MSG_PRIO_HIGH); // this better match ack priority!
```
Comment (jdurgin) [inline src/osd/ReplicatedBackend.cc]: this comment still seems useful

**Comment by jdurgin** [comment]
needs rebase

**Comment by gregsfortytwo** [comment]
I think Josh was right, the comment is still useful so we should leave it, but good to remove that assert!

**Pull request merged**